iqtree2/2.2.2/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:xenial as app
+
+# for easy upgrade later. ARG variables only persist during build time.
+ARG IQTREE2_VER="2.2.2"
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="IQ-Tree2"
+LABEL software.version="2.2.2"
+LABEL description="Efficient software for phylogenomic inference"
+LABEL website="http://www.iqtree.org/"
+LABEL source.code.website="https://github.com/iqtree/iqtree2"
+LABEL license="https://github.com/iqtree/iqtree2/blob/master/LICENSE"
+LABEL maintainer="Jade Wang"
+LABEL maintainer.email="jwang7@health.nyc.gov"
+
+#install dependencies
+RUN apt-get update && apt-get install -y \
+ wget
```

Comment:
We have found that using --no-install-recommends reduces the size of the final image. Could you include it?

It would change lines 19-20 to something like

```
RUN apt-get update && apt-get install -y --no-install-recommends wget && \
 apt-get autoclean && rm -rf /var/lib/apt/lists/*
 ```